pages/landing/index.tsx

```diff
@@ -0,0 +1,125 @@
+import Image from "next/image";
+import styles from "@/pages/landing/styles.module.scss";
+import landing_1 from "@/assets/images/landing/landing_1.png";
+import landing_2 from "@/assets/images/landing/landing_2.png";
+import landing_3 from "@/assets/images/landing/landing_3.png";
+import landing_4 from "@/assets/images/landing/landing_4.png";
+import landing_5 from "@/assets/images/landing/landing_5.png";
+import landing_6 from "@/assets/images/landing/landing_6.png";
+import landing_7 from "@/assets/images/landing/landing_7.png";
+import landing_10 from "@/assets/images/landing/landing_10.png";
+import landing_11 from "@/assets/images/landing/landing_11.png";
+import landing_12 from "@/assets/images/landing/landing_12.png";
+import Ellipse from "@/assets/icons/ic_Ellipse 22.svg";
+
+
+const Landing = () => {
+  return (
+    <>
+      <div className={styles["landing-container"]}>
+        <header className={styles["header"]}>
+          <section className={styles["wikid-banner"]}>
+            <div className={styles["banner-title"]}>
+              <div className={styles["paragraph"]}>남들이 만드는</div>
+              <div className={styles["heading"]}>나만의 위키</div>
+            </div>
+            <div className={styles["btn-wrapper"]}>
+              <button className={styles["wikidMain-page-bts"]}>
+                <strong>위키 만들기</strong>
+              </button>
+            </div>           
```

Comment:
```suggestion
<button className={styles["main-page-button"]}>
```
현재 className에 camelCase, kebab을 섞어서 사용하고 계시는데, 보통은 통일해서 사용합니다! 위에처럼 수정하면 좋지 않을까요?😃 (bts가 버튼을 뜻하는 게 맞다면 button으로 바꾸는 게 좋을 것 같아요)

Reply:
네네! 이부분  통일 시켜야 겠어요!